images/macos/tests/Common.Tests.ps1

```diff
@@ -209,6 +209,10 @@ Describe "Common utilities" {
         "julia --version" | Should -ReturnZeroExitCode
     }
 
+    It "aws-iam-authenticator" {
+        "aws-iam-authenticator --version" | Should -ReturnZeroExitCode

```

Comment:
should be 
` "aws-iam-authenticator version" | Should -ReturnZeroExitCode`

Reply:
you are absolutely right

```
$ aws-iam-authenticator version
{"Version":"v0.5.2","Commit":"292b9b82df69b87af962b92485b254d9f4b10f00"}
```